lib/config.ts

```diff
@@ -0,0 +1,20 @@
+let GRAPHQL_URI = process.env.NEXT_PUBLIC_GRAPHQL_URI as string
+let GRAPHQL_SUBSCRIPTION_URI = process.env.NEXT_PUBLIC_GRAPHQL_SUBSCRIPTION_URI as string
+
```

Comment:
Let's keep the names of the env variables `GRAPHQL_URI` and `GRAPHQL_SUBSCRIPTION_URI`

Reply:
We can't because Next.js requires them to be named like that if we want them exposed to the client. If we name them without the prefix they will only be available on the server side.